geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/executor/sortedset/ZAddExecutor.java

```diff
@@ -77,10 +77,14 @@ private int findAndValidateZAddOptions(Command command, Iterator<byte[]> command
           executorState.chFound = true;
           optionsFoundCount++;
           break;
+        case "-inf":
+        case "inf":
+        case "+inf":
+          scoreFound = true;
+          break;
```

Comment:
It would be good to rebase now that @nonbinaryprogrammer's PR for infinity handling in ZADD is merged.

Reply:
Did that this morning. Hopefully things will be cleaner.